tests/spdx/test_document_utils.py

```diff
@@ -0,0 +1,27 @@
+#  SPDX-FileCopyrightText: Copyright (c) 2023 spdx contributors
```

Comment:
Not sure if this is valid, I would understand the specification to either use `Copyright (c) ..` or `SPDX-FileCopyrightText: 2023..`. The suggested change would align with the header in `test_creation_info_writer.py`.
```suggestion
#  SPDX-FileCopyrightText: 2023 spdx contributors
```

Reply:
changed and adapted my copyright profile (again^^)